org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/BackendUtils.java

```diff
@@ -1385,117 +1385,121 @@ public static void mergeServiceTemplateAinServiceTemplateB(ServiceTemplateId ser
         TTopologyTemplate topologyTemplateA = repository.getElement(serviceTemplateIdA).getTopologyTemplate();
         TServiceTemplate serviceTemplateB = repository.getElement(serviceTemplateIdB);
         TTopologyTemplate topologyTemplateB = serviceTemplateB.getTopologyTemplate();
-        Optional<Integer> shiftLeft = topologyTemplateB.getNodeTemplateOrRelationshipTemplate().stream()
-            .filter(x -> x instanceof TNodeTemplate)
-            .map(x -> (TNodeTemplate) x)
-            .max(Comparator.comparingInt(n -> ModelUtilities.getLeft(n).orElse(0)))
-            .map(n -> ModelUtilities.getLeft(n).orElse(0));
-        if (shiftLeft.isPresent()) {
-            Map<String, String> idMapping = new HashMap<>();
-
-            // collect existing node template ids
-            topologyTemplateB.getNodeTemplateOrRelationshipTemplate().stream()
+        if (topologyTemplateB != null) {
```

Comment:
I would have put the short branch first (so `== null` and then the one liner), but OK since this is an urgend quick fix.